flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ExplainOperation.java

```diff
@@ -24,6 +24,7 @@
  * Operation to describe an EXPLAIN statement. NOTES: currently, only default behavior(EXPLAIN PLAN
  * FOR xx) is supported.
  */
+@Deprecated
```

Comment:
Also add `@deprecated use {@link ExplainDetailsOperation} instead` in the java doc